aten/src/ATen/Declarations.cwrap

```diff
@@ -1574,24 +1553,11 @@
     - THTensor* self
 ]]
 [[
-  name: frac_
-  types:
-    - floating_point
-  backends:
-    - CPU
-    - CUDA
+  name: _frac
```

Comment:
Can we please use `_th_` prefix for all of them? It's insane how inconsistent this is at the moment, and I can't wrap my head around the ping pong between those implementations.

Reply:
Will do